forge-gui/res/lists/TypeLists.txt

```diff
@@ -273,6 +274,7 @@ Thalakos:Thalakoses
 Thopter:Thopters
 Thrull:Thrulls
 Tiefling:Tieflings
+Time-Lord:Time Lords
```

Comment:
Is this dealt with elsewhere in the code to get it to display right?

Reply:
Don't know where to do that. Only this way can I get the IsPresent of TARDIS to work.

Reply:
I assume you tried changing both to "Time Lord" right?

What kind of error do you get?

Reply:
nvm seems to work